apps/api/src/app/widgets/usecases/get-subscriber-preference/get-subscriber-preference.usecase.ts

```diff
@@ -28,6 +31,12 @@ export class GetSubscriberPreference {
       true
     );
 
+    this.analyticsService.track('Fetch User Preferences - [Notification Center]', command.organizationId, {
+      _organization: command.organizationId,
+      _subscriber: command.subscriberId,
```

Comment:
```suggestion
      _subscriber: subscriber._id,
```

We should use only the internal id here, or just remove it all together